pkg/clustermanager/reconcile_logicalcluster.go

```diff
@@ -74,8 +103,9 @@ func (c *controller) processAddLC(newCluster *lcv1alpha1.LogicalCluster) error {
 		return err
 	}
 
-	// Update status to NotReady
+	// Update status Initializing
 	newCluster.Status.Phase = lcv1alpha1.LogicalClusterPhaseInitializing
+	newCluster.Spec.Managed = true
```

Comment:
Can you change the Spec ?

Reply:
Will need to be tested.

Reply:
I think we need to force it on the object creation